round4/team11

```diff
@@ -0,0 +1,97 @@
+# Chrome Extension Idea: Currency Converter
+
+## Authors
+
+Kim Fung, Cherilynn Chow, Sam May, Emre Guler
+
+## Problem Statement
+
+Online shoppers, travelers, and finance enthusiasts often encounter prices listed in foreign currencies when browsing the internet. This can make it challenging to quickly understand the true cost of products, services, or investments in their home currency. Currently, users must either manually convert these currencies using separate apps or websites, which interrupts their browsing experience, or rely on mental approximations, which can be inaccurate.
+
+## Target Audience
+
+The target audience for Currency Converter includes:
+
+- Online shoppers buying from international websites
+- Travelers planning trips or purchases in different countries
+- Financial analysts and investors monitoring international markets
+- Students and professionals who engage with international documents and reports
+- Users needing quick and accurate currency conversions while browsing
+
+## Description
+
+Currency Converter is a Chrome Extension that seamlessly integrates real-time currency conversion into the user's browsing experience. With a hover or click, users can instantly convert prices on any webpage to their home currency or a preferred currency. It eliminates the need to disrupt the browsing experience with manual calculations or external websites, offering a streamlined, in-page conversion feature.
+
+## Selling Points
+
+1. Real-time currency conversion directly on web pages, without navigating away
+2. Customizable preferences for default home currency and frequently used currencies
+3. Automatically detects and highlights prices on web pages for conversion
+4. Supports conversion for a comprehensive list of global currencies, updated in real-time
```

Comment:
It would be good to think about how exactly you want to find updated exchange rates. I think there's good technical merit regardless of if you build a web scraper (Scrapy) or just use an API available.